connector/json-test-connector/src/main.rs

```diff
@@ -1,13 +1,13 @@
 mod source;
 
-use fluvio::{TopicProducer, RecordKey};
+use fluvio::{spu::SpuSocketPool, RecordKey, TopicProducer};
 use fluvio_connector_common::{Source, connector, Result};
 use futures::StreamExt;
 
 use crate::source::TestJsonSource;
 
 #[connector(source)]
-async fn start(config: CustomConfig, producer: TopicProducer) -> Result<()> {
+async fn start(config: CustomConfig, producer: TopicProducer<SpuSocketPool>) -> Result<()> {
```

Comment:
can you create type alias like: 
```
type DefaultTopicProducer = TopicProducer<SpuSocketPool>
```
so it will be be easier to use by rest of client.

Reply:
Much better! I changed to use `DefaultTopicProducer`.

Reply:
I'd maybe suggest TopicProduceProol ?

Reply:
Sure! I rename it to `TopicProducePool`